common_benchmark_suite/openxla/benchmark/models/jax/model_interfaces.py

```diff
@@ -16,11 +17,21 @@ class InferenceModel(abc.ABC, typing.Generic[T, U]):
   """Interface to interact with a JAX inference model."""
 
   @abc.abstractmethod
-  def generate_inputs(self) -> T:
-    """Transforms raw input data into model inputs."""
+  def generate_default_inputs(self) -> Any:
```

Comment:
I feel like we should store raw data in GCS and have a script to fetch them and upload the preprocessed data, which can be fetched later during benchmark.

So this method shouldn't be needed as the raw data are stored in GCS. Maybe we can add a TODO references to #12 and say this will be removed later.

Reply:
Added TODO